docs/reference/ilm/set-up-lifecycle-policy.asciidoc

```diff
@@ -32,28 +46,24 @@ PUT _ilm/policy/my_policy
 }
 ------------------------
 
-<1> Rollover the index when it reaches 25GB in size
-<2> Delete the index when its 30 days old
-
-{ilm} will manage an index using the policy defined in the
-`index.lifecycle.name` index setting. If this setting does not exist in the
-settings for a particular index, {ilm} will not manage that index.
+<1> Roll over the index when it reaches 25GB in size
+<2> Delete the index 30 days after rollover
 
-To set the policy for an index there are two options:
+NOTE: {ilm-init} policies are stored in the global cluster state and can be included in snapshots by  
+setting `include_global_state` parameter to `true`. 
```

Comment:
> setting `include_global_state` parameter to `true`. 

It's not clear which API this parameter belongs to. I'd mention and link to them here.